packages/ui/src/common/components/CKEditor/CKEditor.stories.tsx

```diff
@@ -36,7 +36,7 @@ export const InlineEditor = Template.bind({})
 ClassicEditor.args = {
   minRows: 8,
   maxRows: 20,
-  onChange: (event, editor) => info(editor.getData()),
+  onChange: action('onChange'),
```

Comment:
```suggestion
  onChange: (event, editor) => action('onChange')(editor.getData()),
```
Just one more thing: you can't read the markdown from the editor attributes. But I think the markdown is the only useful information.